README.md

```diff
@@ -239,6 +281,45 @@ Now call the delegate pods in cluster2 from cluster1:
 kubectl --context "$CLUSTER1" run foo -it --rm --image alpine --command -- sh -c "apk add curl && curl nginx"
 ```
 
+#### Adding an additional Target Cluster to an existing Source Cluster
+
+This section assumes you have already installed Admiralty on the source cluster (i.e. cluster1),
+and just want to add an additional target cluster to it; we will keep the cluster2 name for consistency.
+
+Assuming you installed multicluster-scheduler with Helm (v3), you must upgrade it with the same tool.
+
+The easiest way is to retrieve the existing version of the configuration, and append the new cluster name to the targets section.
+
+```
+$ diff -C 2 old_setup.yaml new_setup.yaml | tee add_c2.patch
+*** old_setup.yaml	2020-06-12 09:26:57.000000000 -0700
+--- new_setup.yaml	2020-06-12 09:26:49.000000000 -0700
+***************
+*** 33,34 ****
+--- 33,35 ----
+  targets:
+  - name: catinstall
++ - name: c2
+```
+
+```bash
+helm get values -n admiralty multicluster-scheduler -o yaml > old_setup.yaml
+cp old_setup.yaml new_setup.yaml
+
+# edit or patch new_setup.yaml
+patch new_setup.yaml add_c2.patch 
```

Comment:
Since the user would need new_setup.yaml to create a patch, and I'm not sure we can provide a generic diff-patch, I'd prefer either a description of what needs to be done (add `- name: c2` to the targets list) or a `jq` trick after `helm get values -o json` (instead of `-o yaml`).

Reply:
I did not know how to easily show the needed changes.
I agree it is not ideal.
Would appreciate some suggestions.

Reply:
Converted into a one line as per your suggestion.